R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -385,7 +386,7 @@ test_that("spark.mlp", {
   # Test predict method
   mlpTestDF <- df
   mlpPredictions <- collect(select(predict(model, mlpTestDF), "prediction"))
-  expect_equal(head(mlpPredictions$prediction, 6), c(0, 1, 1, 1, 1, 1))
+  expect_equal(head(mlpPredictions$prediction, 6), c("1.0", "0.0", "0.0", "0.0", "0.0", "0.0"))
```

Comment:
Currently, SparkR ML classification algorithm wrappers supposes all labels are character, so the prediction is character as well. Users can cast it to numeric if it's applicable.